VideoLocker/src/main/java/org/edx/mobile/view/FormFieldSelectFragment.java

```diff
@@ -155,8 +155,8 @@ private void selectCurrentOption() {
             for (int i = 0; i < listView.getCount(); i++) {
                 final FormOption option = (FormOption) listView.getItemAtPosition(i);
                 if (null != option && TextUtils.equals(option.getValue(), currentValue)) {
-                    listView.setSelection(i);
                     listView.setItemChecked(i, true);
+                    listView.setSelection(i);
```

Comment:
Apparently it matters what order these are called in. Just swapping them fixes the scroll issue.